app/helpers/administrate/application_helper.rb

```diff
@@ -1,21 +1,20 @@
 module Administrate
   module ApplicationHelper
-    PLURAL_MANY_COUNT = 2.1
-
     def render_field(field, locals = {})
       locals.merge!(field: field)
       render locals: locals, partial: field.to_partial_path
     end
 
-    def display_resource_name(resource_name)
+    def display_resource_name(resource_name, count = 2.1)
```

Comment:
I see that a constant `PLURAL_MANY_COUNT` was introduced. I kept my solution, but I set the default for the count argument to 2.1.